Error on test.mk when creating dev environment

<!-- Thanks for filing an issue! Before hitting the button, please answer these questions. It's helpful to search the existing GitHub issues first. It's likely that another user has already reported the issue you're facing, or it's a known issue that we're already aware of-->

**Is this a request for help?**: Yes

---

**Is this an ISSUE or FEATURE REQUEST?** (choose one): Issue

---

**What version of acs-engine?**: Current master (commit 5fa24518)

---


<!--
If this is a ISSUE, please:
  - Fill in as much of the template below as you can.  If you leave out
    information, we can't help you as well.

If this is a FEATURE REQUEST, please:
  - Describe *in detail* the feature/behavior/change you'd like to see.

In both cases, be ready for followup questions, and please respond in a timely
manner.  If we can't reproduce a bug or think a feature already exists, we
might close your issue.  If we're wrong, PLEASE feel free to reopen it and
explain why.
-->

**Orchestrator and version (e.g. Kubernetes, DC/OS, Swarm)** N/A


**What happened**: I checked out latest master onto Windows, and ran `.\scripts\devenv.ps1`.  The script terminated in Step 16/17 with the following message:

```
Step 16/17 : RUN make bootstrap
 ---> Running in 56c76f97c64c
make: glide: Command not found
fatal: Not a git repository (or any of the parent directories): .git
fatal: Not a git repository (or any of the parent directories): .git
fatal: Not a git repository (or any of the parent directories): .git
Makefile:128: test.mk: No such file or directory
make: *** No rule to make target 'test.mk'.  Stop.
The command '/bin/sh -c make bootstrap' returned a non-zero code: 2
```

**What you expected to happen**: Expected development container to be built with no errors.  I found that the development container was created successfully up to commit id dd31285e, and the error started occurring with commit id 97ab47bd (resolution for issue #1256).  In that commit I got only one "fatal: not a git repository" error but in current master I get 3 as shown above.  In both I get the makefile `test.mk` error.


**How to reproduce it** (as minimally and precisely as possible):

* Check out commit id 97ab47bd on Windows.
* Run .\scripts\devenv.ps1
* Note errors
* Check out commit id 5fa24518 (master at time of writing) on Windows
* Run .\scripts\devenv.ps1
* Note errors

**Anything else we need to know**:

--- a/Dockerfile
+++ b/Dockerfile
@@ -37,7 +37,7 @@ RUN git clone https://github.com/akesterson/shunit.git /tmp/shunit \
 WORKDIR /gopath/src/github.com/Azure/acs-engine
 
 # Cache vendor layer
-ADD Makefile versioning.mk glide.yaml glide.lock /gopath/src/github.com/Azure/acs-engine/
+ADD Makefile test.mk versioning.mk glide.yaml glide.lock /gopath/src/github.com/Azure/acs-engine/
 RUN make bootstrap
 
 ADD . /gopath/src/github.com/Azure/acs-engine

